internal/pkg/cli/flag.go

```diff
@@ -185,7 +186,7 @@ Must be of the format '<keyName>:<dataType>'.`
 	countFlagDescription         = "Optional. The number of tasks to set up."
 	cpuFlagDescription           = "Optional. The number of CPU units to reserve for each task."
 	memoryFlagDescription        = "Optional. The amount of memory to reserve in MiB for each task."
-	imageFlagDescription         = "Optional. The image to run instead of building a Dockerfile."
+	imageFlagDescription         = "Optional. The docker image to use instead of building from a Dockerfile."
```

Comment:
Should we remove optional here for our init commands? It's technically required if `-d` is not provided.
```suggestion
	imageFlagDescription         = fmt.Sprintf(`The location of an existing Docker image.
Mutually exclusive with -%s, --%s`, dockerFileFlagShort, dockerFileFlag)
```
I wanted to mention "location" somewhere so that the user can relate the flag to the manifest field.